nasa/cmr-metadata-review
# CMRARC-360: The Old Variable Level 2 Science Keyword Field in ECHO10 Records is Being Populated

Originally, this ticket wanted the following:
---------

There are 2 fields for the variable level 2 science keyword in ECHO10 records:

ScienceKeywords / ScienceKeyword / VariableLevel 1Keyword / VariableLevel 2Keyword / Value

ScienceKeywords / ScienceKeyword / VariableLevel 2Keyword / Value  This is the correct field

The variable level 2 keywords are being populated in the old field

-------

Here is a screenshot of the issue. The two columns circled are supposedly duplicates and the ticket is claiming that the second circled column is the correct one. 

<img width="1656" alt="Screenshot 2024-04-18 at 11 58 05 AM" src="https://github.com/nasa/cmr-metadata-review/assets/135638667/6ffbf1b5-f0a1-454e-bfe2-f9c40d8e2e3e">

Upon further inspection, it would seem that this was a misunderstanding. If you use the following curl statement: curl "https://cmr.earthdata.nasa.gov/search/collections.echo10?concept_id=C2111722183-ORNL_CLOUD&pretty=true"

you'll see that the structure looks like the following: 
![Screenshot 2024-04-18 at 1 03 33 PM](https://github.com/nasa/cmr-metadata-review/assets/135638667/313ff7d5-51e0-4ba0-8540-5972afb985e0)

So in conclusion, the first circle column is the correct column and the second circled column is the duplicate. I've removed that duplicate column and as well as another. All columns should now render correctly when users ingest new collections for review. There were also some old console.logs I cleaned up. 

To test:
Ingest and echo10 record on cmr-dahsboard. Any GHRC record should do. Once ingested, select the recrod and click 'See Review Detail then 'See Collection Review Details' then 'Science Keywords'. There should be a column for each of these arrays and no others: 

          ["ScienceKeywords", "ScienceKeyword", "CategoryKeyword"],
          ["ScienceKeywords", "ScienceKeyword", "TopicKeyword"],
          ["ScienceKeywords", "ScienceKeyword", "TermKeyword"],
          ["ScienceKeywords", "ScienceKeyword", "VariableLevel1Keyword", "Value"],
          ["ScienceKeywords", "ScienceKeyword", "VariableLevel1Keyword", "VariableLevel2Keyword", "Value"],
          ["ScienceKeywords", "ScienceKeyword", "VariableLevel1Keyword", "VariableLevel2Keyword", "VariableLevel3Keyword", "Value"],
          ["ScienceKeywords", "ScienceKeyword", "DetailedVariableKeyword"]

![Screenshot 2024-04-18 at 1 10 35 PM](https://github.com/nasa/cmr-metadata-review/assets/135638667/b68467c2-98ac-43a6-90dd-dad4e27f3581)
